.github/workflows/ping-other-repos.yml

```diff
@@ -0,0 +1,25 @@
+name: Remote Dispatch Action Initiator
+
+on:
+  push:
+  repository_dispatch:
```

Comment:
If I understand this correctly, this workflow will run when another repository dispatches _back_ to mdn/browser-compat-data. My guess here is that this was so you could test things, but I'm not sure we need to do this in the final workflow. At very least, it would never actually happen, unless we also provide a PAT to every repository that we dispatch _to_.

Reply:
> If I understand this correctly, this workflow will run when another repository dispatches _back_ to mdn/browser-compat-data. My guess here is that this was so you could test things, but I'm not sure we need to do this in the final workflow.

Yeah, agreed, it’s not necessary — now removed

Reply:
In that case:

```suggestion
```

Reply:
Yup, removed `repository_dispatch`